pkg/sdk/api/v1beta1/node_agent_types.go

```diff
@@ -35,6 +36,7 @@ type NodeAgent struct {
 
 type NodeAgentFluentbit struct {
 	Enabled                 *bool                        `json:"enabled,omitempty"`
+	EnvVars                 []corev1.EnvVar              `json:"envVars,omitempty"`
```

Comment:
This should already be doable using daemonset overrides, I wouldn't add this explicitly here.

To use this with daemonset overrides you just have to specify a subset of a daemonset config with just enough information to allow the merger find the path to the correct container. It should look like this:

```
daemonSet:
  spec:
    template:
      spec:
        containers:
        - name: <containerName>
          env: ...
```

Reply:
In the logging CR
```
spec:
  nodeAgents:
  - name: <name>
    nodeAgentFluentbit:
      daemonSet:
        ...
```